source/common/grpc/json_transcoder_filter.cc

```diff
@@ -70,6 +72,14 @@ class TranscoderImpl : public Transcoder {
   std::unique_ptr<TranscoderInputStream> response_stream_;
 };
 
+bool isGrpcResponse(const Http::HeaderMap& headers) {
+  if (strcmp("200", headers.Status()->value().c_str()) != 0) {
```

Comment:
Please use `getResponseStatus` here to match similar code elsewhere.

Reply:
Done.